api/v1alpha1/common.go

```diff
@@ -0,0 +1,17 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+var (
+	// DisplayNameAnnotationKey is the key of the annotation used to set the display name of the resource
+	DisplayNameAnnotationKey = GroupVersion.Group + "/displayname"
+)
+
+// ComponetVersions Indicates the fields required for a specific version of Component.
+type ComponetVersions struct {
+	Version    string      `json:"version"`
+	AppVersion string      `json:"appVersion"`
+	UpdatedAt  metav1.Time `json:"updatedAt"`
+	CreatedAt  metav1.Time `json:"createdAt"`
+	Digest     string      `json:"digest"`
```

Comment:
Digest for what? Helm charts?

Reply:
https://github.com/kubebb/design/issues/6#issuecomment-1549275564 
According to index.yaml, it contains `version` and `appVersion`.